### PR TITLE
Widen container css

### PR DESCRIPTION
**Pull request by hungnguyen94**
This branch overrides the CSS of the container, widening it to 95% of the screen space.   

Some pages have to be fixed, because they aren't centered correctly. The map-view page doesn't scale correctly, if I'm not mistaken.

### CODE REVIEW

File: src/main/webapp/content/css/custom.css [hunk 1]
```diff
@@ -2,8 +2,20 @@
...
+}
+@media (min-width: 992px) {
+  .container {
+    width: 95%;
+  }
+}
+@media (min-width: 1200px) {
+  .container {
+    width: 95%;
+  }
```
Comment (rubenwiersma) [inline src/main/webapp/content/css/custom.css]: Is it necessary to do this for each size separately?

**Comment by rubenwiersma** [comment]
I've looked through the code.

File: src/main/webapp/content/css/custom.css (hunk 1 above)
Comment (hungnguyen94) [inline src/main/webapp/content/css/custom.css]: Maybe not, but this was copied from the original css.

**Comment by hungnguyen94** [comment]
The centering of the pages still have to be fixed. @cadw

**Comment by rubenwiersma** [comment]
Will merge this, and I'll fix the width of the map in the 'one page scripting' branch.

**Pull request merged**